### ISSUE TITLE
Arabic labels on demo map

### ISSUE THREAD

**Issue by mapmeld** [openmaptiles/www.openmaptiles.org#24]
Arabic letters in map labels are separated and placed left-to-right

Here's how Dubai ( دبي‎ ) appears:

<img width="338" alt="screen shot 2018-07-25 at 3 17 29 pm" src="https://user-images.githubusercontent.com/643918/43222355-ec195c06-901d-11e8-82da-aa94d1f74ce5.png">

Fortunately this is something which has been fixed in MapBoxGL - when you first load this site, you'll see MapBoxGL with Arabic labels (blink and you might miss it - it cycles through the languages) https://mapbox.github.io/mapbox-gl-language/

**Comment by klokan** [comment]
On which URL was the screenshot made?

It is correct at https://www.maptiler.com/maps/#basic//vector/1/0/-20

**Comment by mapmeld** [comment]
wow, I'm pretty sure that it's what I saw yesterday on https://www.maptiler.com/maps/#positron//vector/9.95/55.2966/25.2891 , and I'm checking my web history... it looks fine now so I'll close the issue.  Maybe I was on a weird connection and loaded a font / used a fallback font in an unexpected way.

Burmese still is rendering incorrectly, but that's complicated. Closing this issue for now

<img width="236" alt="screen shot 2018-07-26 at 12 22 41 pm" src="https://user-images.githubusercontent.com/643918/43274996-a8cf5d40-90ce-11e8-91ac-23f37492a2b1.png">
